common/had_map_utils/src/had_map_utils.cpp

```diff
@@ -101,6 +101,7 @@ std::vector<lanelet::BasicPoint3d> resamplePoints(
 
   // Calculate accumulated lengths
   const auto accumulated_lengths = calculateAccumulatedLengths(line_string);
+  if(accumulated_lengths.empty()) return {};
```

Comment:
Function `findNearestIndexPair()` expects at least 2 elements in the `accumulated_lengths`: https://github.com/autowarefoundation/autoware.universe/blob/main/map/lanelet2_extension/lib/utilities.cpp#L121

```suggestion
  if(accumulated_lengths.size() < 2) return {};
```

Reply:
@maxime-clem 
By the way if resample point is one the closest should always be the first one point, why does it needs at least two points?

Reply:
Function `findNearestIndexPair()` contains 2 conditions that assume `accumulated_lengths` contains _at least_ 2 elements.
```C++
  // Front
  if (target_length < accumulated_lengths.at(1)) {
    return std::make_pair(0, 1);
  }

  // Back
  if (target_length > accumulated_lengths.at(N - 2)) {
    return std::make_pair(N - 2, N - 1);
  }
```